docs/baseband/tutorials/getting_started.rst

```diff
@@ -0,0 +1,317 @@
+.. _getting_started:
+
+***************
+Getting Started
+***************
+
+This tutorial covers the basic features of Baseband.  It assumes that Numpy and
+the Astropy Units module have been imported::
+
+    >>> import numpy as np
+    >>> import astropy.units as u
+
+.. _getting_started_reading:
+
+Reading Files
+=============
+
+Opening Files
+-------------
+
+Each format supported by Baseband has a master input/output function,
+accessible by importing the corresponding format module. For example, to read
+the sample VDIF file that comes with Baseband (sample files can all be found in
+the `baseband.data` module)::
+
+    >>> import baseband.vdif as vdif
+    >>> from baseband.data import SAMPLE_VDIF
+    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
+
+To close the file::
+
+    >>> fh.close()
+
+The same syntax can be used to open a file of any format.  To open Baseband's
+sample DADA file, for example::
+
+    >>> from baseband import dada
+    >>> from baseband.data import SAMPLE_DADA
+    >>> fh_dada = dada.open(SAMPLE_DADA, 'rs')
+    >>> dada_data = fh_dada.read()
+    >>> fh_dada.close()
+
+In general, file I/O and data manipulation use the same syntax across all
+file formats.  When using ``open`` for Mark 4 and Mark 5B files, however, two
+keywords - ``ntrack``, and ``decade`` - may need to be set manually.  For these
+and VDIF, ``frames_per_second`` may also need to be passed if it can't be read
+or inferred from the file. Notes on such features and quirks of individual
+formats can be found within the :ref:`Specific file format
+<specific_file_formats_toc>` section.
+
+For now, let's go back to using VDIF files.
+
+Decoding Data and the Sample File Pointer
+-----------------------------------------
+
+We gave `~baseband.vdif.open` the ``'rs'`` flag, which opens the file in 
+"stream reader" mode.  The function returns an instance of
+`~baseband.vdif.base.VDIFStreamReader`, a wrapper around `io.BufferedReader`
+that adds methods to decode files as data frames and seek to and read individual
+data samples.  To read the first 12 data samples into an `~numpy.ndarray`, we
+would use the `~baseband.vdif.base.VDIFStreamReader.read` method::
+
+    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
+    >>> d = fh.read(12)
+    >>> type(d)
+    <... 'numpy.ndarray'>
+    >>> d.shape
+    (12, 8)
+    >>> d[:, 0].astype(int)  # First thread.
+    array([-1, -1,  3, -1,  1, -1,  3, -1,  1,  3, -1,  1])
+
+The sample VDIF file has 8 concurrent frequency bands, or "channels", which are
+mapped as the columns of the array.
+
+We can access information from the header by printing ``fh``::
```

Comment:
The information is not all from the header. Maybe just "We can get information about the file by printing ``fh``::"

Reply:
Agreed, and changed.